mlir-sycl/include/mlir/Dialect/SYCL/Utils/Utils.h

```diff
@@ -37,8 +37,10 @@ SYCLRangeGetOp createSYCLRangeGetOp(Type resTy, TypedValue<MemRefType> range,
                                     Location loc);
 
 /// Construct sycl.id with id type \p idTy and indexes \p indexes.
-TypedValue<MemRefType> constructSYCLID(IDType idTy, ArrayRef<Value> indexes,
-                                       OpBuilder builder, Location loc);
+TypedValue<MemRefType> createSYCLIDConstructorOp(IDType idTy,
```

Comment:
Probably a matter of personal preference, but I liked the old name (`constructSYCLID`) of the function better. I'll leave it up to you to decide on the name though.

Reply:
TBH, I also like the old name better, but this is more inline with what we can find in [`llvm::IRBuilderBase`](https://llvm.org/doxygen/classllvm_1_1IRBuilderBase.html). Up to Whitney, I guess.

Reply:
I will keep the changed function, as it is more inline with `IRBuilderBase`, and also other functions we have in this file.